polkadot/node/network/statement-distribution/src/lib.rs

```diff
@@ -350,57 +252,38 @@ impl<R: rand::Rng> StatementDistributionSubsystem<R> {
 					.await?;
 				},
 				StatementDistributionMessage::NetworkBridgeUpdate(event) => {
-					// pass all events to both protocols except for messages,
+					// pass all events to all protocols except for messages,
```

Comment:
we should remove this logic which forwarded the message to the legacy or v2 subsystems. I don't think we'll get a new version of the statement-distribution subsystem anytime soon